framework/src/state_machine/constants.ts

```diff
@@ -17,3 +17,6 @@ export const EVENT_INDEX_FINALIZE_GENESIS_STATE = Buffer.from([1]);
 
 export const EVENT_INDEX_BEFORE_TRANSACTIONS = Buffer.from([2]);
 export const EVENT_INDEX_AFTER_TRANSACTIONS = Buffer.from([3]);
+
+export const EVENT_TRANSACTION_NAME = 'transaction';
+export const EVENT_REWARD_MINTED_DATA_NAME = 'Reward Minted Data';
```

Comment:
if we use only `EVENT_REWARD_MINTED_DATA` then it will be confused with actual event data so its better to use full name `EVENT_REWARD_MINTED_DATA_NAME `

Reply:
Same thought, going to keep it unchanged.